### PR TITLE
Added RecyclerView to Pending

### PR DESCRIPTION
**Pull request by SinadjanRudge**
Updated Item_PatientAppointments_Rudge: Pending now has recyclerview which shows Appointments
                                                                       CardView has added buttons

### CODE REVIEW

File: app/src/main/java/com/triadss/doctrack2/activity/patient/fragment/AppointmentPending.java [hunk 1]
```diff
@@ -1,27 +1,70 @@
...
+import java.util.List;
 
 public class AppointmentPending extends Fragment {
 
+
+    RecyclerView recyclerView;
+    private AppointmentRepository appointmentRepository;
     private BottomNavigationView bottomNavigationView, PatientbottomNavigationView;
 
+   ArrayList<String> courseName = new ArrayList<>(Arrays.asList("Data Structure"));
```
Comment (RevyaS) [inline app/src/main/java/com/triadss/doctrack2/activity/patient/fragment/AppointmentPending.java:31]: This is an appointment Pending page, please use appropriate variable names.
Also why is there Data Structure as its initial data?

File: app/src/main/java/com/triadss/doctrack2/activity/patient/fragment/PatientAppointmentAdapter.java [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+import java.util.ArrayList;
+
+// Extends the Adapter class to RecyclerView.Adapter
+// and implement the unimplemented methods
+public class PatientAppointmentAdapter extends RecyclerView.Adapter<PatientAppointmentAdapter.ViewHolder> {
+    ArrayList  courseName;
+    Context context;
+
+    // Constructor for initialization
+    public PatientAppointmentAdapter(Context context,  ArrayList courseName) {
```
Comment (RevyaS) [inline app/src/main/java/com/triadss/doctrack2/activity/patient/fragment/PatientAppointmentAdapter.java:25]: This is an appointment Pending page, please use appropriate variable names.

File: app/src/main/res/layout/list_item.xml [hunk 3]
```diff
@@ -25,58 +25,33 @@
...
-            android:layout_marginStart="30dp"
-            android:layout_marginTop="24dp"
-            android:layout_marginEnd="33dp"
-            android:layout_marginBottom="8dp"
+            android:layout_marginStart="35dp"
+            android:layout_marginTop="44dp"
+            android:layout_marginEnd="20dp"
+            android:layout_marginBottom="53dp"
             android:layout_toEndOf="@+id/button"
             android:text="Button" />
```
Comment (RevyaS) [inline app/src/main/res/layout/list_item.xml:56]: Please rename the Button text to its appropriate name even for placeholder purposes

File: app/src/main/res/layout/list_item.xml [hunk 4]
```diff
@@ -25,58 +25,33 @@
...
+            android:layout_below="@+id/nametext"
             android:layout_alignParentStart="true"
             android:layout_alignParentBottom="true"
-            android:layout_marginStart="15dp"
-            android:layout_marginTop="23dp"
-            android:layout_marginBottom="8dp"
+            android:layout_marginStart="39dp"
+            android:layout_marginTop="44dp"
+            android:layout_marginBottom="53dp"
             android:text="Button" />
```
Comment (RevyaS) [inline app/src/main/res/layout/list_item.xml:42]: Please rename the Button text to its appropriate name even for placeholder purposes

File: app/src/main/java/com/triadss/doctrack2/activity/patient/fragment/AppointmentPending.java (hunk 1 above)
Comment (SinadjanRudge) [reply app/src/main/java/com/triadss/doctrack2/activity/patient/fragment/AppointmentPending.java]: Changed {courseName} to {Pending}

File: app/src/main/java/com/triadss/doctrack2/activity/patient/fragment/PatientAppointmentAdapter.java (hunk 2 above)
Comment (SinadjanRudge) [reply app/src/main/java/com/triadss/doctrack2/activity/patient/fragment/PatientAppointmentAdapter.java:25]: Changed {courseName} to {Pending}

File: app/src/main/res/layout/list_item.xml (hunk 3 above)
Comment (SinadjanRudge) [reply app/src/main/res/layout/list_item.xml:56]: Created new XML file called item_list_pending with the appropriate names for widgets

File: app/src/main/res/layout/list_item.xml (hunk 4 above)
Comment (SinadjanRudge) [reply app/src/main/res/layout/list_item.xml:42]: Created new XML file called item_list_pending with the appropriate names for widgets

**Comment by RevyaS** [comment]
Approved

**Pull request merged**